lib/Event.php

```diff
@@ -27,121 +28,121 @@ class Event extends ApiResource
+    const CUSTOMER_DELETED                         = 'customer.deleted';
+    const CUSTOMER_UPDATED                         = 'customer.updated';
+    const CUSTOMER_DISCOUNT_CREATED                = 'customer.discount.created';
+    const CUSTOMER_DISCOUNT_DELETED                = 'customer.discount.deleted';
+    const CUSTOMER_DISCOUNT_UPDATED                = 'customer.discount.updated';
+    const CUSTOMER_SOURCE_CREATED                  = 'customer.source.created';
+    const CUSTOMER_SOURCE_DELETED                  = 'customer.source.deleted';
+    const CUSTOMER_SOURCE_EXPIRING                 = 'customer.source.expiring';
+    const CUSTOMER_SOURCE_UPDATED                  = 'customer.source.updated';
+    const CUSTOMER_SUBSCRIPTION_CREATED            = 'customer.subscription.created';
+    const CUSTOMER_SUBSCRIPTION_DELETED            = 'customer.subscription.deleted';
+    const CUSTOMER_SUBSCRIPTION_TRIAL_WILL_END     = 'customer.subscription.trial_will_end';
+    const CUSTOMER_SUBSCRIPTION_UPDATED            = 'customer.subscription.updated';
+    const FILE_CREATED                             = 'file.created';
+    const INVOICE_CREATED                          = 'invoice.created';
+    const INVOICE_DELETED                          = 'invoice.deleted';
+    const INVOICE_FINALIZED                        = 'invoice_finalized';
+    const INVOICE_MARKED_UNCOLLECTIBLE             = 'invoice.marked_uncollectible';
+    const INVOICE_PAYMENT_FAILED                   = 'invoice.payment_failed';
+    const INVOICE_PAYMENT_SUCCEEDED                = 'invoice.payment_succeeded';
+    const INVOICE_SENT                             = 'invoice.sent';
+    const INVOICE_UPCOMING                         = 'invoice.upcoming';
+    const INVOICE_UPDATED                          = 'invoice.updated';
+    const INVOICE_VOIDED                           = 'invoice.voided';
+    const INVOICEITEM_CREATED                      = 'invoiceitem.created';
+    const INVOICEITEM_DELETED                      = 'invoiceitem.deleted';
+    const INVOICEITEM_UPDATED                      = 'invoiceitem.updated';
+    const ISSUING_AUTHORIZATION_CREATED            = 'issuing_authorization.created';
+    const ISSUING_AUTHORIZATION_UPDATED            = 'issuing_authorization.updated';
+    const ISSUING_CARD_CREATED                     = 'issuing_card.created';
+    const ISSUING_CARD_UPDATED                     = 'issuing_card.updated';
+    const ISSUING_CARDHOLDER_CREATED               = 'issuing_cardholder.created';
+    const ISSUING_CARDHOLDER_UPDATED               = 'issuing_cardholder.updated';
+    const ISSUING_DISPUTE_CREATED                  = 'issuing_dispute.created';
+    const ISSUING_DISPUTE_UPDATED                  = 'issuing_dispute.updated';
+    const ISSUING_SETTLEMENT_CREATED               = 'issuing_settlement.created';
+    const ISSUING_SETTLEMENT_UPDATED               = 'issuing_settlement.updated';
```

Comment:
Please remove those two for now, they should not be public (fixing on our end too).